main.cpp

```diff
@@ -215,7 +215,7 @@ int main()
     HAL_UART_Receive_DMA(&huart2, (uint8_t *)&command, sizeof(command));
 
     for (;;) {
-        HAL_Delay(DELAY_IN_MAIN_LOOP); //delay in ms
+        HAL_Delay(1); //delay in ms
```

Comment:
Is there a reason why this exact value was chosen (and the define removed)? Consider adding a comment here.